.vscode/settings.json

```diff
@@ -18,5 +18,9 @@
   },
   "java.configuration.updateBuildConfiguration": "automatic",
   // LSP was ooming and it recommended this change
-  "java.jdt.ls.vmargs": "-XX:+UseParallelGC -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xmx2G -Xms100m -Xlog:disable"
+  "java.jdt.ls.vmargs": "-XX:+UseParallelGC -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xmx2G -Xms100m -Xlog:disable",
+  "java.test.config": {
+    "vmargs": [ "-Dstripe.disallowGlobalResponseGetterFallback=true"]
```

Comment:
This got pulled in via https://github.com/stripe/stripe-java/pull/1880 - do we know why we needed this?

Reply:
this is needed to get one of our unit tests to run in vs code.  see https://github.com/stripe/stripe-java/pull/1875